src/test/java/com/google/sps/MealServletTest.java

```diff
@@ -249,6 +256,31 @@ public void getIdOfSimilarTest() throws IOException, ServletException {
         assertEquals(expected, actual);
     }
 
+    // Get a similar meal from Datastore
+    // Expected result: Long in JSON format with id of second meal.
+    @Test
+    public void returnSameTypeTest() throws IOException, ServletException {
+        DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+        ds.put(createMealEntity(PIZZA_1));
+        ds.put(createMealEntity(PIZZA_2));
+        ds.put(createMealEntity(MEAL_1));
+        
+        MealServlet servlet = new MealServlet();
+        MockHttpServletRequest request = new MockHttpServletRequest();
+        MockHttpServletResponse response = new MockHttpServletResponse();
+        request.setPathInfo("/similar");
+        request.addParameter("id", PIZZA_1.getId().toString());
+        servlet.doGet(request, response);
+
+        // returnIdOfSimilar() should  return id of the same category meel
+        Long similarId = PIZZA_2.getId();
+        
```

Comment:
Remove this empty line.

Too many empty lines makes the code harder to read, as it is artificially spread out. Empty lines should be used to split code like splitting text into paragraphs. In tests we usually split into setup, action and expectations (and this split is achieved in this test by the other 2 empty lines).